spec/hardware/iommu.fmf

```diff
@@ -0,0 +1,26 @@
+summary: |
+    Select or provision a guest with the `Input–output memory management unit`.
+
+description: |
+    .. code-block::
+
+       iommu:
+           # Boolean, whether the guest supports IOMMU
+           is-supported: "x"
```

Comment:
A boolean key should be documented with `true|false` value, see e.g. https://github.com/teemtee/tmt/blob/main/spec/hardware/virtualization.fmf#L18